IdentityCore/src/controllers/MSIDRequestControllerFactory.m

```diff
@@ -35,6 +35,11 @@
 #import "MSIDRequestParameters+Broker.h"
 #import "MSIDAuthority.h"
 #import "MSIDSignoutController.h"
+#if TARGET_OS_OSX
+#import "MSIDXpcSilentTokenRequestController.h"
+#endif
+
+static NSString *newXpcController = @"new_xpc_controller";
```

Comment:
nit: Should this be static const NSString *newXpcController?

Reply:
Good catch, this is not used , removed